src/Components/Checkout.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import React from "react";
+
+function Checkout() {
+ 
+const [firstname, setFirstName] = useState("");
+const [lastname, setLastName] = useState("");
+const [email, setEmail] = useState("");
+const [zipcode, setZipCode] = useState("");
+
+function submitPurchase(){
+    if(firstname && lastname && email && zipcode){
+alert("You have adopted birds. Thank you!")
+
+    }
+    console.log(submitPurchase)
+}
```

Comment:
This code will never run. You are passing the specs for this feature but your alert never shows up, and the form isn't reset but the whole page is reloaded (Not something we want to happen in one-page web apps!). To get full points for this you will need to make the change and pass all the specs with it.